wiki/Commands/give/index.md

```diff
@@ -0,0 +1,39 @@
+---
+title: .give Command
+layout: wiki
+category: Command
+---
+Gives you an item with custom NBT settings in creative.
+
+## Syntax
+- `.give <item name> <amount> <metadata> <NBT data>`
```

Comment:
This syntax is outdated. See https://github.com/Wurst-Imperium/Wurst-Client/blob/master/Wurst%20Client/src/tk/wurst_client/commands/GiveCmd.java#L25